dist/bin/kctf-challenge

```diff
@@ -241,7 +241,35 @@ function kctf_chal_debug_ssh {
   require_cluster_config
   COMMAND="debug ssh" parse_container_name $@ || return
 
-  "${KCTF_BIN}/kubectl" exec "deployment/${CHALLENGE_NAME}" --namespace "${CHALLENGE_NAMESPACE}" -c "${CONTAINER}" -it -- /bin/bash
+  pods=($("${KCTF_BIN}/kubectl" get pods -o jsonpath='{.items[*].metadata.name}'))
+
+  if [[ ${#pods[@]} -eq 0 ]]; then
+    _kctf_log_err 'No pods found. Is the challenge running?'
+    return 1
+  fi
+
+  pod="${pods[0]}"
+  if [[ ${#pods[@]} -ne 1 ]]; then
+    _kctf_log "Found ${#pods[@]} pods, connecting to the most recent one."
+    _kctf_log "To connect to another pod, run 'kubectl exec pod/PODNAME --namespace ${CHALLENGE_NAMESPACE} -c ${CONTAINER} -it -- /bin/bash'"
```

Comment:
I thought about this too but decided not to add it since the list of pod names is not really useful to choose from.
I added a note now how to list the pods.